lib/src/helpers/default_options_list.dart

```diff
@@ -1,24 +1,46 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_sfsymbols/flutter_sfsymbols.dart';
 import 'package:help_ukraine_widget/help_ukraine_widget.dart';
 
 ///
 const defaultOptionsList = [
   HelpOptionButton(
     title: 'Donate',
-    icon: SFSymbols.money_dollar,
     iconSize: 20,
+    child: Text(
+      '💸',
```

Comment:
Turn these emojis into constants?

Reply:
By the way, since we're using emoji instead of icons, we could probably remove the `flutter_sfsymbols` dependency.